app/models/invoice.rb

```diff
@@ -9,6 +9,10 @@ class Invoice < ApplicationRecord
   after_create :cancel_previous_invoices
   default_scope { order(created_at: :desc) }
 
+  def self.latest
+    reorder(created_at: :desc).first
+  end
```

Comment:
Perfect!

Reply:
Great, the rule of what is "latest" is defined in the model.

I have to admit, at first this code looked strange. But I can see that it's right 👍 
However now I'm looking, I think we could make it simpler.

```suggestion
  def self.latest
    reorder(:created_at).last
  end
```
But really it doesn't matter.

🤔 It feels like `first` and `last` should be able to accept an ordering parameter. But I checked, they don't.